SWLOR.Game.Server/Feature/UsePerkFeat.cs

```diff
@@ -273,6 +273,11 @@ void CompleteActivation(string id, float abilityRecastDelay)
             CheckForActivationInterruption(activationId, position);
             SetLocalInt(activator, activationId, (int)ActivationStatus.Started);
 
+            if (GetIsPC(activator) && !Space.IsPlayerInSpaceMode(activator))
+            {
+                ApplyEffectToObject(DurationType.Temporary, EffectCutsceneImmobilize(), activator, 1.5f);
```

Comment:
Any real significance to 1.5 seconds? Do you want to account for heavy armor here? I.E longer immobilize time compared to light?

Reply:
For abilities with cast times 1.5 seconds is usually the lowest they go (pretty sure that's what throw lightsaber is at). 1.5 seconds should be enough time for, if someone is trying to kite an enemy who is right behind them, for them to both be stopped and for the enemy chasing them to begin a flurry of attacks.